libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -379,6 +403,19 @@ export type TigerSpecificFunctions = {
     ) => Promise<IInvitationUserResponse>;
     getWorkspaceDataFiltersLayout?: () => Promise<WorkspaceDataFiltersLayout>;
     setWorkspaceDataFiltersLayout?: (workspaceDataFiltersLayout: WorkspaceDataFiltersLayout) => Promise<void>;
+    getWorkspaceDataFilters?: (workspaceId: string, objectId: string) => Promise<WorkspaceDataFiltersResult>;
+    setWorkspaceDataFilters?: (
+        workspaceId: string,
+        workspaceDataFilter: WorkspaceDataFilter,
+    ) => Promise<WorkspaceDataFiltersResult>;
+    getWorkspaceDataFilterSettings?: (
+        workspaceId: string,
+        objectId: string,
+    ) => Promise<WorkspaceDataFilterSettingsResult>;
+    setWorkspaceDataFilterSettings?: (
```

Comment:
Please rename `setWorkspaceDataFilterSettings` → `setWorkspaceDataFilterSetting` (without any `s` after `Setting`).

`setWorkspaceDataFilterSettings` can make people think that the function can set an array of workspace data filter settings, can't it?